discovery/gossiper.go

```diff
@@ -256,6 +256,10 @@ type Config struct {
 	// here?
 	AnnSigner lnwallet.MessageSigner
 
+	// ScidCloser is an instance of gossipScidCloser that helps the
+	// gossiper cut down on spam channel announcements.
```

Comment:
Nit: be more detailed in the comment that the spam channel announcements refer to chan announcements of already closed channels ?